9.2.6 Release

su-sws-jira opened the pull request:
# DO NOT DELETE

Commit:
STN-856:  Accordion credit caption overflow bug (#951)

* fix(STN-856): accordion caption credit overflow bug fix

su-sws-jira committed:
9.2.6

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr956-3wvl9wluovbjjclc31jg5yalg0bwhjhf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61300fbf08cf47f73c5b034b

Visual Diffs:
- https://dashboard.tugboat.qa/61300fbf08cf47f73c5b034b/visualdiffs

Commit:
Fix(STN-863): Full-width Color Band extends beyond the content region on mobile. (#957)

* fix: adding overflow-x: hidden to html selector as overflow-x: hidden on
the body tag ( mobile browsers ) isn't necessarily respected.

The color band component is able to be displayed full-width and it is
currently relying on overflow-x: hidden on the body to prevent overflow
issues, but after testing on mobile devices you can see that this simply
doesn't work.

Add overflow-x: hidden; to html selector and this seems to resolve the
issue.

* chore: compile css

Commit:
Fix(STN-842): postcard uniform height in collections fix (#953)

* fix(STN-842): postcard uniform height in collections fix

Postcards contained within collections were not getting the uniform
height specified by the --uniform-height modifier

- Target .field-hs-collection-items > .hb-card with 100% height.
- Target .field-hs-collction-items > .ptype-hs-postcard with display:
  flex

* chore: compile css

su-sws-jira committed:
Updated dependencies Sep 8 2021

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr956-3wvl9wluovbjjclc31jg5yalg0bwhjhf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61300fbf08cf47f73c5b034b

Visual Diffs:
- https://dashboard.tugboat.qa/61300fbf08cf47f73c5b034b/visualdiffs

pookmish committed:
Sparkbox Sprint 48

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr956-3wvl9wluovbjjclc31jg5yalg0bwhjhf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61300fbf08cf47f73c5b034b

Visual Diffs:
- https://dashboard.tugboat.qa/61300fbf08cf47f73c5b034b/visualdiffs

pookmish committed:
HSD8-1142 changed combined filter to separate filters on the user list page

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr956-3wvl9wluovbjjclc31jg5yalg0bwhjhf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61300fbf08cf47f73c5b034b

Visual Diffs:
- https://dashboard.tugboat.qa/61300fbf08cf47f73c5b034b/visualdiffs

pookmish committed:
Enable duplicate for private page components

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr956-3wvl9wluovbjjclc31jg5yalg0bwhjhf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61300fbf08cf47f73c5b034b

Visual Diffs:
- https://dashboard.tugboat.qa/61300fbf08cf47f73c5b034b/visualdiffs

pookmish committed:
HSD8-1144 Replace ";" with "|" for d7 people research area importer

pookmish committed:
HSD8-1136 Reduce contextual links for non-admins

pookmish committed:
HSD8-1123 Allow course importer url form to be emptied

pookmish committed:
HSD8-1140 Enable dblog staging sites

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr956-3wvl9wluovbjjclc31jg5yalg0bwhjhf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61300fbf08cf47f73c5b034b

Visual Diffs:
- https://dashboard.tugboat.qa/61300fbf08cf47f73c5b034b/visualdiffs

pookmish committed:
HSD8-1146 Add statistics redirect for statweb

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr956-3wvl9wluovbjjclc31jg5yalg0bwhjhf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61300fbf08cf47f73c5b034b

Visual Diffs:
- https://dashboard.tugboat.qa/61300fbf08cf47f73c5b034b/visualdiffs

joegl committed:
HSD8-1149: Provision music
